pydantic/schema.py

```diff
@@ -516,6 +516,27 @@ def model_type_schema(
     return out_schema, definitions, nested_models
 
 
+def enum_process_schema(enum: EnumMeta) -> Dict[str, Any]:
+    """
+    Take a single ``enum`` and generate its schema.
+
+    This is similar to the ``model_process_schema`` function, but applies to ``Enum`` objects.
+    """
+    from inspect import getdoc
+
+    schema: Dict[str, Any] = {'title': enum.__name__}
+
+    # Python assigns all enums a default docstring value of 'An enumeration', so
+    # all enums will have a description field even if not explicitly provided.
+    schema['description'] = getdoc(enum)
+
+    # Add enum values and the enum field type to the schema.
+    schema.update({'enum': [item.value for item in cast(Iterable[Enum], enum)]})
```

Comment:
```suggestion
    schema['enum'] = [item.value for item in cast(Iterable[Enum], enum)]
```
or you could writing the entire thing to:

```py
schema = {
    'title': enum.__name__,
    # comment ...
    'description': getdoc(enum),
    # comment ...
    'enum': ...,
}
```

Reply:
Well that was silly. Fixed!